Examples/Swift/ShapeCollectionFeatureExample.swift

```diff
@@ -0,0 +1,70 @@
+import Mapbox;
+
+let MapboxAccessToken = "<# Access Token #>"
+
+@objc(ShapeCollectionFeatureExample_Swift)
+
+class ShapeCollectionFeatureExample_Swift: UIViewController, MGLMapViewDelegate {
+    
+    var mapView : MGLMapView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        mapView = MGLMapView(frame: view.bounds)
+        mapView.styleURL = MGLStyle.lightStyleURL(withVersion: 9)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        
+        mapView.setCenter(CLLocationCoordinate2D(latitude:38.897435, longitude: -77.039679), zoomLevel: 12, animated: false)
+        
+        mapView.delegate = self
+        view.addSubview(mapView)
+    }
+    
+    func mapView(_ mapView: MGLMapView, didFinishLoading style: MGLStyle) {
+        
+        // Parse the GeoJSON data.
+        DispatchQueue.global().async {
+            let url = URL(string: "https://api.mapbox.com/datasets/v1/mapbox/cj004g2ay04vj2xls3oqdu2ou/features?access_token=\(MapboxAccessToken)")
```

Comment:
Agreed. It would be great to show how you can use the Datasets editor and Datasets API with the iOS SDK. I think that can be a priority once we have more DDS examples!